language/move-model/src/model.rs

```diff
@@ -54,7 +54,7 @@ use move_binary_format::{
 use move_core_types::{
     account_address::AccountAddress, identifier::Identifier, language_storage, value::MoveValue,
 };
-use move_symbol_pool::Symbol as StringSymbol;
+use move_symbol_pool::Symbol as MoveStringSymbol;
```

Comment:
Cool, this is a better name I think 👍

Reply:
Yeah I thought it was more clear as it started being used more